packages/hebao_v1/contracts/modules/dapps/LRCStakingModule.sol

```diff
@@ -37,12 +37,20 @@ contract LRCStakingModule is SubAccountDAppModule
     address public lrcTokenAddress;
 
     constructor(
-        Controller       _controller,
-        IUserStakingPool _stakingPool,
-        address          _lrcTokenAddress
+        Controller       _controller
         )
         public
         SecurityModule(_controller)
+    {
+    }
+
+    function init(
+        address wallet,
+        IUserStakingPool _stakingPool,
+        address          _lrcTokenAddress
+        )
+        external
+        onlyFromWalletOwner(wallet)
```

Comment:
`onlyFromWalletOwner(wallet)` doesn't seem to make much sense?

Reply:
Sorry, I had a misunderstanding about it. I had made this contract Ownable and use onlyOwner modifier for it.